toast-tk-dao/src/main/java/io/toast/tk/dao/service/dao/access/repository/RepositoryDaoService.java

```diff
@@ -53,6 +53,7 @@ public RepositoryDaoService(
 		this.eDaoService = eDaoServiceFactory.create(dbName);
 	}
 
+	@Deprecated
 	public String getRepoAsJson() {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS1133)